scripts/recreate_records

```diff
@@ -63,7 +63,7 @@ inspirehep db drop --yes-i-know
 inspirehep db create
 inspirehep index destroy --force --yes-i-know
 inspirehep index init
+inspirehep fixtures init
```

Comment:
@jacquerie We would need to run this extra CLI command during CI testing as well. Would importing and calling it in conftests be the way?